index.html

```diff
@@ -0,0 +1,173 @@
+            </div>
+        </div>
+    </section>
+    <section id="program" class="flex-col">
+        <h2>Main Program</h2>
+        <hr class="dash">
+        <div class="program-items">
+            <div>
+                <img src="asset/speaker.svg" alt="speaker">
+                <h6>Lecture</h6>
+                <p>
+                    Listen to the speakers from various countries about the messages of sharing and opening.
+                </p>
+            </div>
+            <div>
+                <img src="asset/exhibit.svg" alt="speaker">
+                <h6>OV Exhibit</h6>
+                <p>
+                    Listen to the speakers from various countries about the messages of sharing and opening.
+                </p>
+            </div>
+            <div>
+                <img src="asset/forum.svg" alt="speaker">
+                <h6>Forum</h6>
+                <p>
+                    get opportunities to network with CC affiliates around the world, also after the summit.
+                </p>
+            </div>
+            <div>
+                <img src="asset/workshop.svg" alt="speaker">
+                <h6>Workshop</h6>
+                <p>
+                    Listen to the speakers from various countries about the messages of sharing and opening.
+                </p>
+            </div>
+            <div>
+                <img src="asset/network.svg" alt="speaker">
+                <h6>OG Ignite</h6>
+                <p>
+                    get opportunities to network with CC affiliates around the world, also after the summit.
+                </p>
+            </div>
+        </div>
+        <button type="button" id="join-btn" class="btn">Join OV Gaming Summit 2015</button>
+        <a href="#">See the whole program</a>
+    </section>
+    <section id="speaker-section" class="flex-col">
+        <div class="flex-col">
+            <h2>Featured Speakers</h2>
+            <hr class="dash">
+        </div>
+        <div id="speaker-card-container"></div>
+    </section>
+    <section id="desktop-sponsors">
+        <div id="sponsor-wrapper">
+            <h3>Partner</h3>
+            <hr class="dash">
+            <div>
+                <img class="sponsors-img" src="asset/logitech.png" alt="logitech">
+                <img src="asset/redbullcom-logo.svg" alt="redbull">
+                <img src="asset/steam.png" alt="steam">
+                <img src="asset/riot.png" alt="riot">
+                <img class="sponsors-img" src="asset/twitch.png" alt="twitch">
+            </div>
+        </div>
+    </section>
+    <footer id="footer-desktop" class="footer-section">
+        <div class="footer-container flex-row">
+            <div class="logo-img-container">
+                <img src="asset/fire.png" alt="logo">
+                <h2>
+                    OG Gaming
+                    <br> 
+                    Summit 2023
+                </h2>
+            </div>
```

Comment:
- [ ] The position of this div containing logo in your footer is not identical to the behance design. kindly consider adding margin-right to it to move it further left to make it identical.
- [ ] And also consider applying this change on About page as well.